Client/Frontend/TabBar/SearchViewController.swift

```diff
@@ -64,13 +71,13 @@ extension SearchViewController: UITableViewDataSource {
     }
 
     func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return searchEngines?.count ?? 0
+        return searchEngines?.list.count ?? 0
     }
 }
 
 extension SearchViewController: UITableViewDelegate {
     func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
-        let engine = searchEngines?[indexPath.item]
```

Comment:
Same here, I think this should be `indexPath.row` .. It may work, but officially `item` is for the `UICollectionView`.